var/spack/repos/builtin/packages/r3d/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class R3d(MakefilePackage):
+    """Fast, robust polyhedral intersections, analytic integration, and
+    conservative voxelization."""
+
+    homepage = "https://github.com/devonmpowell/r3d"
+    url      = "https://github.com/devonmpowell/r3d.git"
+
+    version('181219', git=url, commit='47308f68c782ed3227d3dab1eff24d41f6421f21')
+
+    variant("test",  default=False, description="Build R3D regression tests")
+
+    def install(self, spec, prefix):
+
+        make_args = [
+            'CC={0}'.format(spack_cc),
+        ]
+
+        make('libr3d.a', *make_args)
+
+        # R3D does not have an install target so create our own here.
+        mkdirp(prefix.include)
+        _my_headers = find('.', '*.h', recursive=False)
```

Comment:
No need for leading underscores, variables within functions are already private.

Reply:
Removed